.vscode/settings.json

```diff
@@ -1,5 +1,8 @@
 {
   "python.formatting.provider": "black",
   "jest.disabledWorkspaceFolders": ["api"],
-  "jest.rootPath": "pro"
+  "jest.rootPath": "pro",
+  "python.analysis.extraPaths": [
+    "./api/src"
+  ]
```

Comment:
Cette modification n'est pas liée au reste du commit. Si elle est vraiment nécessaire, il faut en faire un commit à part.

Pourquoi souhaites-tu ajouter cette ligne ? Je suppose que c'est parce que VSCode ne trouve pas la bibliothèque `pcapi` ? Si c'est ça, je pense que que ça n'est pas la bonne façon de faire. VSCode est [capable de détecter](https://code.visualstudio.com/docs/python/environments#_how-the-extension-looks-for-environments) l'environnement virtuel utilisé, et si tu as `pcapi` installé dans cet environnement virtuel (ce qui devrait être le cas), tu ne devrais donc pas avoir besoin d'ajouter cette option. Je te laisse voir cela avec des utilisateurs de VSCode (ce que je ne suis pas).